scripts/vagrant.sh

```diff
@@ -30,3 +30,11 @@ chown -R ${VAGRANT_USER}:${VAGRANT_USER} ${VAGRANT_HOME}/.ssh
 
 echo '==> Recording box config date'
 date > /etc/vagrant_box_build_time
+
+echo '==> Set DNS to known non interupting defaults'
+# See https://wiki.ipfire.org/dns/public-servers for details
+# Chosen servers are Digitalcourage e.V. and Cloudflare
+cat << EOF > /etc/resolv.conf
```

Comment:
why not 8.8.8.8? ;)

Reply:
AFAIK theoretically they aren't public. :) Just that nobody changed that yet in fear of breaking the web. In the ipfire link you can find the result of the firewall and router os vendor with the same name about how to address that fact. Google works as well but I can't remember if they ever stated an intent to make these servers publicly available.